advocacy_docs/pg_extensions/wait_states/installing.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Installing EDB Wait States
+navTitle: Installing
+---
+
+## On Linux 
+
+To install EWSBW on a Linux system:
```

Comment:
Should this be EWSBW or just EDB Wait States?

Reply:
We should use EDB Wait States and never EWSBW.

Reply:
We can use the BGW abbreviation for the background worker on subsequent references after saying EDB Wait States Background Worker (BGW).